strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/PyPiArtifactCoordinateTest.java

```diff
@@ -0,0 +1,28 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+/**
+ * @author Gregory Reshetniak
+ *
+ */
+public class PyPiArtifactCoordinateTest
+{
+
+    @Test
+    public void testArtifactPathToCoordinatesConversion()
```

Comment:
Could you please add some more tests for this based on real-world packages. Try and find some odd examples and check if it works.

Reply:
Sure, will add that.